v3/spring-cloud-starter-aws-v3-metrics/pom.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2013-2019 the original author or authors.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xmlns="http://maven.apache.org/POM/4.0.0"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<artifactId>v3</artifactId>
+		<groupId>io.awspring.cloud</groupId>
+		<version>2.4.1-SNAPSHOT</version>
+	</parent>
+	<artifactId>spring-cloud-starter-aws-v3-metrics</artifactId>
```

Comment:
Something for discussion - should not we call this module `cloudwatch` instead of `metrics`? Similar to how `messaging` module covered SQS and SNS because these were the only options back then for messaging in AWS, i believe we should have now SQS and SNS modules.

Reply:
Cloudwatch is a service which involves logs, metrics so in this case since the support it is specific to metrics I think make sense to keep it that way. The reasoning for spring-cloud-gcp metrics module was similar.

Reply:
I remain unconvinced but let it be ;-)